app/Casts/TagArrayCast.php

```
@@ -8,48 +8,53 @@
+class TagArrayCast implements CastsAttributes
 {
 	/**
 	 * @param Model               $model      the associated model class
 	 * @param string              $key        the name of the SQL column holding the stringified array
 	 * @param mixed               $value      the stringified array
 	 * @param array<string,mixed> $attributes all SQL attributes of the entity
 	 *
-	 * @return array<int,string> the array
+	 * @return array<int,Tag> the array
 	 */
 	public function get(Model $model, string $key, mixed $value, array $attributes): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$model'. (Unused Code Rules)

(UnusedFormalParameter)